src/vs/workbench/common/editor/editorGroup.ts

```diff
@@ -332,7 +332,26 @@ export class EditorGroup extends Disposable {
 
 			// More than one editor
 			if (this.mru.length > 1) {
-				this.setActive(this.mru[1]); // active editor is always first in MRU, so pick second editor after as new active
+				const tabClosingOrder = this.configurationService.getValue('workbench.editor.tabClosingOrder');
+				if (tabClosingOrder === 'rtl') {
```

Comment:
@SimonEggert instead of repeating `this.setActive...` so many times, can we just store the editor as variable and then call this method last with the right one?

Reply:
I adjusted the code to avoid multiple `this.setActive`-calls.